lib/gitlab_net.rb

```diff
@@ -30,7 +30,7 @@ def check_access(cmd, repo, actor, changes)
     if resp.code == '200'
       GitAccessStatus.create_from_json(resp.body)
     else
-      GitAccessStatus.new(false, "API is not accesible")
+      GitAccessStatus.new(false, "API is not accessible")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.